changelog/cve-2020-16846.fixed

```diff
@@ -0,0 +1 @@
+CVE-2020-16804 - Prevent shell injections in netapi ssh client
```

Comment:
Just noticed a typo on the CVE reference: CVE-2020-168**4**6

Reply:
Good catch, I'll fix this.